google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/PartitionLookupUtils.java

```diff
@@ -32,9 +32,7 @@ private PartitionLookupUtils() {}
   public static int numPartitions(TopicPath topic) throws StatusException {
     try (AdminClient client =
         AdminClient.create(
-            AdminClientSettings.newBuilder()
-                .setRegion(topic.location().location().region())
-                .build())) {;
+            AdminClientSettings.newBuilder().setRegion(topic.location().region()).build())) {;
```

Comment:
Please remove the spurious trailing semi-colon.

Reply:
Done.